detoks/src/main/java/nl/tudelft/trustchain/detoks/DeToksCommunity.kt

```diff
@@ -3,20 +3,43 @@ package nl.tudelft.trustchain.detoks
 import android.content.Context
 import android.util.Log
 import nl.tudelft.ipv8.Community
+import nl.tudelft.ipv8.IPv4Address
 import nl.tudelft.ipv8.Overlay
 import nl.tudelft.ipv8.Peer
+import nl.tudelft.ipv8.android.IPv8Android
+import nl.tudelft.ipv8.attestation.trustchain.*
+import nl.tudelft.ipv8.attestation.trustchain.store.TrustChainStore
```

Comment:
Looks like a redundant import since ...attestation.trustachain.* is already imported

Reply:
It seems like the import is required, although im not sure why, removed the other redudant imports though